### ISSUE TITLE
Share a method to knit to pdf (if you need it like me!)

### ISSUE THREAD

**Issue by jinfei1125** [macss-modeling/General-Questions#12]
I have been bothered by the problem of knitting to pdf for a while... Sometimes R studio can knit my Rmarkdown to pdf successfully (I can knit the demo successfully), but sometimes I got the following error message:
```
! Package inputenc Error: Unicode char ▇ (U+2587)
(inputenc)                not set up for use with LaTeX.

Error: LaTeX failed to compile ps2-q1.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See ps2-q1.log for more info.
Execution halted
``` 
I guess the reason is that my keyboard is Chinese, so sometimes I accidentally type a Chinese character like a blank space in my Rmd, and it refuses to work. But finding a single Chinese character in the long rmd is too difficult. I have found the [solution](https://stackoverflow.com/questions/58263640/trouble-with-using-latex-with-r-markdown) online.

To sum up, first, make sure install tinytex package using:
```
install.packages("tinytex")
tinytex::install_tinytex()
```
Then change the header of the markdown file to
```
output:
  pdf_document:
    latex_engine: xelatex
```

And then it works!

**Comment by Yilun0221** [comment]
Hi Jinfei,

I have experienced the trouble you mentioned.  In my situation, the saving path included Chinese and I just changed a path with all English words and it worked. I don't know whether it can solve your problem.
